community/sponsors/sponsor-list.json

```diff
@@ -85,45 +74,34 @@
       "bio": "",
       "link": "https://www.savant.com/",
       "image": "savant.jpg"
-    }
-  ],
-  "bronze": [
```

Comment:
Here - this will throw off the JSON file and maybe prevent the website from rendering